blog/arquillian-core-1.4.0.Final.textile

```diff
@@ -0,0 +1,28 @@
+---
+date: 2018-02-27
+---
+
+p. <i class="icon-star"></i> **Extended expression language in Arquillian configuration**
+
+p. In @arquillian.xml@ file, properties are always defined explicitly, as a system property @${myproperty}@ or as an environment variable @${env.myenvironment}@. But in both cases values are set before test execution.
+
+p. With this release, you can implement your own way of resolving values, not just system properties or environment variables.
+
+p. You need to register a new Arquillian service in @ExtensionBuilder@ and it must conform following signature:
```

Comment:
It might be good to either link to the actual implementation so mention how one can really do that. I think this is a bit too brief for anyone who has no familiarity with Arquillian extensions.